extension.ts

```diff
@@ -227,6 +232,54 @@ async function createWebApp(context?: AzureImageTagNode | DockerHubImageTagNode)
   } else if (result.status === 'Cancelled') {
     throw new UserCancelledError();
   }
+  const website = wizard.createdWebSite;
+  let appUri = `https://${website.name}.scm.azurewebsites.net/docker/hook`;
+
+  if (context instanceof AzureImageTagNode) {
+    await createWebhookForWebApp(context, wizard, appUri);
+
+  } else {
+    // point to dockerhub to create a webhook
+    // http://cloud.docker.com/repository/docker/<registryName>/<repoName>/webHooks
+    const dockerhubPrompt: string = "Copy webapp endpoint and head to dockerhub";
+    let response: string = await vscode.window.showInformationMessage("Please head to your dockerhub account to set up a CI/CD webhook", dockerhubPrompt);
+    if (response) {
+      // tslint:disable-next-line:no-unsafe-any
+      clipboardy.writeSync(appUri);
+      // tslint:disable-next-line:no-unsafe-any
+      opn(`https://cloud.docker.com/repository/docker/${context.userName}/${context.repositoryName}/webHooks`);
```

Comment:
Use new vscode open API

Reply:
Telemetry of whether they did this or not.

Reply:
I'm making changes to opn in this branch: https://github.com/Microsoft/vscode-docker/tree/ps/opnToExternal

Just thought it would make sense to standardize this across the code-base.